ci-operator/step-registry/ibmcloud/provision/resourcegroup/ibmcloud-provision-resourcegroup-commands.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# IBM Cloud CLI login
+function ibmcloud_login {
+    export IBMCLOUD_CLI=ibmcloud
+    export IBMCLOUD_HOME=/output   
+    region="${LEASED_RESOURCE}"
+    export region
+    "${IBMCLOUD_CLI}" config --check-version=false
+    echo "Try to login..." 
+    "${IBMCLOUD_CLI}" login -r ${region} --apikey @"${CLUSTER_PROFILE_DIR}/ibmcloud-api-key"
+}
+
+function create_resource_group() {
+    local rg="$1"
+    echo "create resource group ... ${rg}"
+    "${IBMCLOUD_CLI}" resource group-create ${rg} || return 1
+    "${IBMCLOUD_CLI}" target -g ${rg} || return 1
+}
+
+ibmcloud_login
+
+rg_name="${NAMESPACE}-${UNIQUE_HASH}"
```

Comment:
how about adding '-rg', to distinguish the resource group which auto-created?

Reply:
fixed.